### ISSUE TITLE
When opening a file its filename is evaluated as command instead of as file

### ISSUE THREAD

**Issue by vovcacik** [fman-community/fman#372]
This one is tricky. Fman is using cmd's `start` to open files. This is wrong because `start` is supposed to be invoked on other executables:

```batch
START ["title"] [/D path] [/I] [/MIN] [/MAX] [/SEPARATE | /SHARED]
      [/LOW | /NORMAL | /HIGH | /REALTIME | /ABOVENORMAL | /BELOWNORMAL]
      [/NODE <NUMA node>] [/AFFINITY <hex affinity mask>] [/WAIT] [/B]
      [command/program] [parameters]
```

It works in some scenarios and doesn't in others. Alternative would be to use `explorer "file.txt"` command instead. Or some API call.

Scenario:
- lets say I create empty file `touch.exe powershell`
- double click in fman will open the PowerShell terminal
- it is same as calling `start "powershell"` on cmd which will sken the `%PATH%` and `%PATHEXT%` and eventually executes `C:\Windows\System32\WindowsPowerShell\v1.0\powershell.EXE`

Expected behaviour is that explorer will ask me what program I want to open the file with (as it is missing extension).

[fman#372 using cmd-start instead of explorer to invoke files.zip](https://github.com/fman-users/fman/files/1938691/fman.372.using.cmd-start.instead.of.explorer.to.invoke.files.zip)

**Comment by vovcacik** [comment]
This is related to #369 but both are different bugs. However both can be fixed by not calling `start "filename.txt"` internally.

**Comment by mherrmann** [comment]
Fixed in [fman 1.1.4](https://fman.io/changelog#v1.1.4).
